auth 4.4.0-beta1 tarball needs venv

<!-- Hi! Thanks for filing an issue. It will be read with care by human beings. Can we ask you to please fill out this template and not simply demand new features or send in complaints? Thanks! -->
<!-- Also please search the existing issues (both open and closed) to see if your report might be duplicate -->
<!-- Please don't file an issue when you have a support question, send support questions to the mailinglist or ask them on IRC (https://www.powerdns.com/opensource.html) -->

<!-- Tell us what is issue is about -->
 - Program: Authoritative
 - Issue type: Bug report

### Short description
<!-- Explain in a few sentences what the issue/request is -->

### Environment
<!-- Tell us about the environment -->
 - Operating system: Debian unstable
 - Software version: 4.4.0-beta1
 - Software source: source tarball

### Steps to reproduce
<!-- Tell us step-by-step how the issue can be triggered. Please include your configuration files and any (Lua) scripts that are loaded. -->
1. observe pdns/api-swagger.json exists in the tarball
2. try a debian build, with python3 installed, but python3-venv missing

### Expected behaviour
Should just build without venv.

### Actual behaviour

```
make[3]: Entering directory '/<<PKGBUILDDIR>>/pdns'
/usr/bin/python3 -m venv .venv
protoc --cpp_out=./ dnsmessage.proto
The virtual environment was not created successfully because ensurepip is not
available.  On Debian/Ubuntu systems, you need to install the python3-venv
package using the following command.

    apt-get install python3-venv

You may need to use sudo with that command.  After installing the python3-venv
package, recreate your virtual environment.

Failing command: ['/<<PKGBUILDDIR>>/pdns/.venv/bin/python3', '-Im', 'ensurepip', '--upgrade', '--default-pip']

make[3]: *** [Makefile:4053: .venv] Error 1
make[3]: Leaving directory '/<<PKGBUILDDIR>>/pdns'
make[2]: *** [Makefile:575: all-recursive] Error 1
make[2]: Leaving directory '/<<PKGBUILDDIR>>'
make[1]: *** [Makefile:507: all] Error 2
make[1]: Leaving directory '/<<PKGBUILDDIR>>'
```

### Other information

This patch from Habbie makes the problem go away:

```patch
diff --git a/pdns/Makefile.am b/pdns/Makefile.am
index b9cb53256..0e86ea7e7 100644
--- a/pdns/Makefile.am
+++ b/pdns/Makefile.am
@@ -87,7 +87,7 @@ if HAVE_VENV
 	.venv/bin/pip install -U pip setuptools setuptools-git wheel
 	.venv/bin/pip install -r requirements.txt

-api-swagger.json: api-swagger.yaml .venv
+api-swagger.json: api-swagger.yaml
 	.venv/bin/python convert-yaml-to-json.py $< $@
 else # if HAVE_VENV
 if !HAVE_API_SWAGGER_JSON
```

This one does not:

```patch
diff --git a/pdns/Makefile.am b/pdns/Makefile.am
index b9cb53256..0e86ea7e7 100644
--- a/pdns/Makefile.am
+++ b/pdns/Makefile.am
@@ -87,7 +87,7 @@ if HAVE_VENV
 	.venv/bin/pip install -U pip setuptools setuptools-git wheel
 	.venv/bin/pip install -r requirements.txt

-api-swagger.json: api-swagger.yaml .venv
+api-swagger.json: api-swagger.yaml | .venv
 	.venv/bin/python convert-yaml-to-json.py $< $@
 else # if HAVE_VENV
 if !HAVE_API_SWAGGER_JSON
```

--- a/pdns/Makefile.am
+++ b/pdns/Makefile.am
@@ -82,12 +82,10 @@ api-swagger.yaml: $(wildcard ../docs/http-api/swagger/authoritative-api-swagger.
 	cp $< $@
 
 if HAVE_VENV
-.venv: requirements.txt
+api-swagger.json: api-swagger.yaml requirements.txt
 	$(PYTHON) -m venv .venv
 	.venv/bin/pip install -U pip setuptools setuptools-git wheel
 	.venv/bin/pip install -r requirements.txt
-
-api-swagger.json: api-swagger.yaml .venv
 	.venv/bin/python convert-yaml-to-json.py $< $@
 else # if HAVE_VENV
 if !HAVE_API_SWAGGER_JSON

